packages/hub/services/gas-estimation.ts

```diff
@@ -96,17 +78,18 @@ export default class GasEstimationService {
     // since transferAmount will be calculated with fee percentage.
     // And gasPrice is price per unit of gas, so it will be multiplied by the gas execution.
     let transferAmount = '1000';
-    let gasPrice = '100';
+    let gasPrice = '1';
     let gas;
+    let usdTokenAddress = await getAddress('usdStableCoinToken', provider);
```

Comment:
Can you add a code comment where we explain that estimations will actually vary between different token implementations but we are using usdc to make things simpler.